test/invariant/fuzz/helpers/IStdCheats.sol

```diff
@@ -35,14 +35,14 @@ interface IStdCheats {
     // Sets an address' balance
     function deal(address who, uint256 newBalance) external;
 
+    // Sets an address' token balance
+    function deal(address token, address to, uint256 amount) external;
```

Comment:
watchout, this is medusa's cheatcode, I have the feeling this `deal` isn't supported (ie loading the balanceOf storage slot, etc) -> the fuzzERC20 token we use exposes mint instead *and* I've removed any limitation of transfer from the test contract (ie to fund, just use `token.transfer(dest, amount)`, without any active prank before)

Reply:
(that's why I kept it as a separate interface instead of reusing forge's one, some cheatcode aren't supported, they just silently revert)

Reply:
Correct, I used mint after this failed, and forgot to delete it